include/bringauto/modules/mission_module/devices/AutonomyDevice.hpp

```diff
@@ -90,6 +93,10 @@ class AutonomyDevice {
 	static MissionModule::AutonomyCommand generateCommand(std::vector<MissionModule::Station> stops, std::string route,
 														  MissionModule::AutonomyCommand::Action action);
 
+	/**
+	 * @brief Map of last sent status timestamps for each device type
+	 */
+	inline static std::map<unsigned int, std::chrono::milliseconds> last_sent_status_timestamps_ {};
```

Comment:
Do not use INLIE static in header files!

Reply:
Explain what is firrefence betwen inline static in header file and inline static in c/cpp files.

Reply:
is no longer inline